iFixit/react-commerce
# Replace warranty banner logo

closes #492 

## Changelog

- Replace logo
- Improve style

## QA

1. Open Vercel preview
2. Visit Parts product list
3. Verify that the warranty banner logo looks like this
    ![Screenshot 2022-07-26 at 16 10 20](https://user-images.githubusercontent.com/4640135/181026851-09220110-5a65-4448-b861-c56613981d13.png)
